tools/generator-go-sdk/internal/generator/templater_models.go

```diff
@@ -292,7 +234,109 @@ func (c modelsTemplater) methods(data GeneratorData) (*string, error) {
 	return &output, nil
 }
 
-func (c modelsTemplater) structLineForField(fieldName, fieldType string, fieldDetails models.SDKField, data GeneratorData) (*string, error) {
+func (c modelsTemplater) structLinesForModel(data GeneratorData, fieldNames []string, excludeComments, excludeDiscriminatedParentMembers bool) (*[]string, error) {
+	output := make([]string, 0)
+
+	for _, fieldName := range fieldNames {
+		fieldDetails := c.model.Fields[fieldName]
+
+		topLevelObject := helpers.InnerMostSDKObjectDefinition(fieldDetails.ObjectDefinition)
+		if excludeDiscriminatedParentMembers && topLevelObject.Type == models.ReferenceSDKObjectDefinitionType {
+			if referencedModel, ok := data.models[*topLevelObject.ReferenceName]; ok && referencedModel.IsDiscriminatedParentType() {
+				// Skip fields that reference a parent model (i.e. an interface rather than a struct)
+				continue
+			}
+		}
+
+		fieldTypeName := "FIXME"
```

Comment:
Is there any context on this?

Reply:
This is a holdover, we should probably be erroring here rather than outputting `FIXME` strings, I'll look to fix that up 👍